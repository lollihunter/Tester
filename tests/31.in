5 20 20
1 1
3 8
4 13
1 8
5 4
5 6
3 2
5 3
2 12
3 9
2 9
3 6
5 15
3 17
5 13
5 16
3 11
4 9
3 3
5 20
