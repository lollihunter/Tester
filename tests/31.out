10
1 19 12 2 10 17 15 13 14 20 
